src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.UnitTests/Queries/GetBulkUploadValidationErrors/BulkUploadValidateDataHandlerTests.cs

```diff
@@ -55,7 +58,7 @@ public BulkUploadValidateDataHandlerTestsFixture()
                 _bulkUploadParser = new Mock<IBulkUploadFileParser>();
                 _bulkUploadParser.Setup(x => x.GetCsvRecords(It.IsAny<long>(), It.IsAny<IFormFile>()))
                     .Returns(new System.Collections.Generic.List<Web.Models.Cohort.CsvRecord>());
-                BulkUploadValidateDataHandler = new FileUploadValidateDataHandler(_outerApiService.Object, _modelMapper.Object, _bulkUploadParser.Object);
+                BulkUploadValidateDataHandler = new FileUploadValidateDataHandler(_outerApiService.Object, _modelMapper.Object, _bulkUploadParser.Object, _authorizationService.Object);
```

Comment:
This is a tiny bit picky (as the existing tests aren't great), but we have added this new property but I can't see if we test that this is set correctly when passing it to the ValidateBulkUploadRequest()